Tests/scripts/prepare_private_content_packs_for_testing.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
```

Comment:
not sure why we need another file like this, why can't we configure it all the same way?
the build shouldn't install private content anyway so it doesn't matter

Reply:
There are 2 ways to do this - either make all the env variables parameters, or save every single env variable in github actions with the same name as the equivalent circle env var.
Neither of those feels right to me - do you have an idea for a better way?
Also applies to `run_private_build_tests`.
@yaakovi would love your opinion here as well

Reply:
as I commented here:
https://github.com/demisto/content/pull/8741#discussion_r480474726
no need to extract all variables, only a few (2-4).

Reply:
After looking into the logic of this script, this isn't just a matter of variables.
The whole logic of working with the buckets here is different.
Changing it would just mean using a lot of booleans and if statements, and every change to the public one might break the private build.
Your call @ronykoz @yaakovi